INSTALL.md

```diff
@@ -9,10 +9,12 @@ Start by creating a database
 
 ```
 sudo -u postgres createuser -s $USER
-createdb gis
+createdb -E UTF8 -l en_GB.UTF8 gis
 ```
 
-Enable PostGIS and hstore extensions with
+The character encoding scheme to be used in the database is *UTF8* and the adopted collation is *en_GB.UTF8*.
```

Comment:
We don't need to change this text, and what are the implications of changing the collation?

Reply:
Collation is not really changed, but just set explicitly, as the default might not always be `’en_GB.UTF8’`. In some installations it might be `'C'` by default for instance and in some other ones defaulted to other locales (like `’it_IT.UTF8’`). In theory it might affect indexes, sort ordering and comparisons within SQL statements in *project.mml*.

Reply:
I'm not aware of any cases where it matters. The only ordering by user-supplied strings that is done is in the last term of an order by clause to get a consistent ordering across metatiles. The collation doesn't matter for this.